services/httpd/handler.go

```diff
@@ -583,6 +585,8 @@ func (h *Handler) serveQuery(w http.ResponseWriter, r *http.Request, user meta.U
 			h.httpError(rw, "error authorizing query: "+err.Error(), http.StatusForbidden)
```

Comment:
Do we want a log event in the omitted else branch to log the fact we can't get a fineAuthorizer?  Seems like something we would want to report, independently of returning it as a StatusForbidden error.

Reply:
Done